packages/core/src/media/metrics/metrics.ts

```diff
@@ -426,7 +429,13 @@ export function addMediaMetricsToStore<TElement, TMediaStream>(
     let key: keyof RawMetrics;
     for (key in metrics.current) {
       const val = metrics.current[key];
-      if (val !== metrics?.previous?.[key]) {
+
+      const shouldSendValue =
+        typeof val === 'number'
+          ? Number.isFinite(val) && !Number.isNaN(val) && val >= 0
```

Comment:
I wonder if we ever do want to send `0` values for any metrics.

Reply:
Yeah it's just >= so filtering any possible negatives